spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -52,6 +52,41 @@
 	</properties>
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
```

Comment:
I have already added this [here](https://github.com/spring-cloud/spring-cloud-dataflow/blob/9efd42b55ac8ade992a7b96c240ea1234b48f4cf/spring-cloud-dataflow-parent/pom.xml#L145-L171)